lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -344,21 +344,39 @@ mixin _WheelEventListenerMixin on _BaseAdapter {
     }
 
     final List<ui.PointerData> data = <ui.PointerData>[];
-    _pointerDataConverter.convert(
-      data,
-      change: ui.PointerChange.hover,
-      timeStamp: _BaseAdapter._eventTimeStampToDuration(event.timeStamp!),
-      kind: ui.PointerDeviceKind.mouse,
-      signalKind: ui.PointerSignalKind.scroll,
-      device: _mouseDeviceId,
-      physicalX: event.clientX.toDouble() * ui.window.devicePixelRatio,
-      physicalY: event.clientY.toDouble() * ui.window.devicePixelRatio,
-      buttons: event.buttons!,
-      pressure: 1.0,
-      pressureMax: 1.0,
-      scrollDeltaX: deltaX,
-      scrollDeltaY: deltaY,
-    );
+    if (event.ctrlKey) {
+      _pointerDataConverter.convert(
```

Comment:
It would be nice if we could decide what combination of key + scroll does what at the framework level, in case some developers have different opinions about that.  Is that something that we could make possible?  I get that we need to prevent the browser from zooming in at the engine level though.